packages/fx-core/src/plugins/solution/fx-solution/arm.ts

```diff
@@ -966,6 +966,9 @@ async function persistBicepTemplates(
     const parameterEnvFolderPath = path.join(projectaPath, configsFolder);
     await fs.ensureDir(parameterEnvFolderPath);
     for (const env of envListResult.value) {
+      if (env === environmentManager.getLocalEnvName()) {
```

Comment:
In previous implementation, we only expect remote environments to be listed by using the  `listEnvConfigs` API.

Now, the `local` environment also be included in the result of this API, it may cause code break in other places that doesn't expect local environment to be in the list.

Reply:
Thanks for the advice, I will pay attention to this. For now, if we filter local out of ```listEnvConfigs``` result, it will cause ```checkEnvExist``` fail. And treeView already handles local scenario. I will check more in later prs.

Reply:
We can have two API: `listAllEnvConfig` and `listRemoteEnvConfig`, and replace the original `listEnvConfig` with `listRemoteEnvConfig`. For `checkEnvExist` API implementation, we can call `listAllEnvConfig`.

Reply:
Good idea, already updated.